src/services/APIHandler.js

```diff
@@ -0,0 +1,154 @@
+export const RequestType = {
```

Comment:
Liked the way you are handling the API calls by writing reusable code for the same purpose

Reply:
Thanks.